### PR TITLE
add additional imu type support [CPP-922]

### PR DESCRIPTION
**Pull request by adrian-kong**
handles additional imu type MurataScha634D03 and TdkIam20680Hp

### CODE REVIEW

File: console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs [hunk 1]
```diff
@@ -97,18 +98,25 @@ impl AdvancedImuTab {
...
+            Ok(imu) => {
+                match imu {
+                    ImuType::BoschBmi160 => {
+                        self.imu_temp = 23_f64 + msg.temp as f64 / 512_f64;
+                    }
+                    ImuType::StMicroelectronicsAsm330Llh => {
+                        self.imu_temp = 25_f64 + msg.temp as f64 / 256_f64;
+                    }
+                    ImuType::MurataScha634D03 => {
+                        self.imu_temp = 25_f64 + msg.temp as f64 / 256_f64;
```
Comment (adrian-kong) [inline console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:111]: for these `imu_temp` values, how should they be handled? (compared to the above)

**Review by silverjam — APPROVED** [review]

File: console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs [hunk 2]
```diff
@@ -97,18 +98,25 @@ impl AdvancedImuTab {
...
+                    }
+                    ImuType::TdkIam20680Hp => {
+                        self.imu_temp = 25_f64 + msg.temp as f64 / 256_f64;
```
Comment (adrian-kong) [inline console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:114]: same as above^

File: console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs (hunk 1 above)
Comment (adrian-kong) [reply console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:111]: @silverjam they seem to have different imu_temp handling formulas (not sure how they were derived)

Comment (silverjam) [reply console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:111]: cc @woodfell ^^

@adrian-kong -- if it's not documented to be different in the SBP manual, I'd leave it the same as the others

Comment (adrian-kong) [inline console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:111]:
```suggestion
                        self.imu_temp = 25_f64 + msg.temp as f64 / 30_f64;
```

File: console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs (hunk 2 above)
Comment (adrian-kong) [reply console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:114]: ```((TEMP_OUT – RoomTemp_Offset)/Temp_Sensitivity) + 25degC```
seems per  https://invensense.tdk.com/wp-content/uploads/2021/11/DS-000409-IAM-20680HP-v1.2-Typ.pdf
needs multiple params?

Comment (silverjam) [reply console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs:114]: Looks like those are defined here (section 3.3.2):

image

Comment (adrian-kong) [reply console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs]: hmmm yea was thinkin but number doesnt seem that nice compared to the others... guess theres no other choice 😢

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -73,6 +73,8 @@ jobs:
         run: |
           if [ "$RUNNER_OS" == "Linux" ]; then
               sudo apt-get update && sudo apt-get install -y capnproto libudev-dev libssl-dev
+              sudo apt install ttf-mscorefonts-installer
```
Comment (silverjam) [inline .github/workflows/main.yml:76]: Trying to tackle this here too: https://github.com/swift-nav/swift-toolbox/pull/1000/commits/e9c49ce232434bd87368cb0485e51e77d70508e3

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4571526174)

**Pull request merged**